docs/SSL/lavalink-with-ssl.md

```diff
@@ -11,3 +11,11 @@ Port : 443
 Password : "www.freelavalink.ga"
 Secure : true
 ```
+Hosted by @ [](https://dsc.gg/extinctop)
+```bash
+Id : Main (If Required)
```

Comment:
Id part is unnecessary https://lavalink.darrennathanael.com/FAQ/frequently-asked-questions/